_layouts/api.html

```diff
@@ -82,6 +82,11 @@ <h2>
     <p>No self-service</p>
   {% endunless %}
 
+  {% if page.more_languages_by_request  == true %}
+    <hr/>
+    <p>➕ More languages available per request.</p>
```

Comment:
I'd probably move this to right next to where we list the languages

<img width="772" alt="Screenshot 2023-03-29 at 00 50 36" src="https://user-images.githubusercontent.com/11457984/228363719-e9c4f180-346b-49d5-89e4-407143ac12e5.png">

Reply:
Maybe `by request` is probably better than `upon request`, just to keep it consistent with the code